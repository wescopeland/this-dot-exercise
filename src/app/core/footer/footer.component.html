<footer class="tde-footer">
  <div class="container-fluid">
    <!-- this is really just meant to provide some color balance at the bottom of the screen -->
  </div>
</footer>