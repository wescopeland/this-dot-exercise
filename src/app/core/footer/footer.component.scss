.tde-footer {
  position: fixed;
  bottom: 0;
  width: 100%;
  height: 24px;
  line-height: 24px;
  background-color: #263238;
  color: white;
}
