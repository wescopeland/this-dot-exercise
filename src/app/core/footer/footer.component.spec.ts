import { Spectator } from '@netbasal/spectator';
import { createTestComponentFactory } from '@netbasal/spectator/jest';

import { FooterComponent } from './footer.component';

describe('Component: FooterComponent', () => {
  let spectator: Spectator<FooterComponent>;
  const createComponent = createTestComponentFactory<FooterComponent>({
    component: FooterComponent,
    detectChanges: false,
    declarations: [],
    imports: [],
    componentProviders: []
  });

  beforeEach(() => {
    spectator = createComponent();
  });

  it('exists', () => {
    expect(spectator.component).toBeDefined();
  });

  it('should render the footer', () => {
    expect(spectator.fixture).toMatchSnapshot();
  });
});
