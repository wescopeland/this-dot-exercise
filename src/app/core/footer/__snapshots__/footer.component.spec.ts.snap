// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: FooterComponent should render the footer 1`] = `
<tde-footer>
  <footer
    class="tde-footer"
  >
    <div
      class="container-fluid"
    />
  </footer>
</tde-footer>
`;
